### PR TITLE
Fix remaining typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S CHANGES.md,./ASWF/tsc-meetings -L ba,dout,halfs,iif,leapyear,lits,nnumber,offsetp`

Signed-off-by: luz paz <luzpaz@github.com>

### CODE REVIEW

**Comment by luzpaz** [comment]
Found these typos after the fact. Sorry for all the noise.  
Note: there is a source fix in [b240ff1](https://github.com/AcademySoftwareFoundation/openexr/pull/1272/commits/b240ff16e9e9ed5cc50935602e2cdd06381f5037). Please review closely.

File: src/lib/OpenEXR/ImfRle.cpp [hunk 1]
```diff
@@ -51,7 +51,7 @@ rleCompress (int inLength, const char in[], signed char out[])
         else
         {
             //
-            // Uncompressable run
+            // Uncompressible run
```
Comment (cary-ilm) [inline src/lib/OpenEXR/ImfRle.cpp]: Should be incompressible

File: src/lib/OpenEXRCore/internal_rle.c [hunk 2]
```diff
@@ -42,7 +42,7 @@ internal_rle_compress (
         }
         else
         {
-            /* uncompressable */
+            /* uncompressible */
```
Comment (cary-ilm) [inline src/lib/OpenEXRCore/internal_rle.c]: Should be incompressible

**Review by cary-ilm — COMMENTED** [review]
These all look good, thanks for the cleanup. Two minor suggestions: "uncompressible" is not a word, the proper term is "incompressible", could you kindly make that change?

The code change is fine, it's simply a forward function declaration.

I did not know about codespell, very handy!

File: src/lib/OpenEXRCore/internal_rle.c (hunk 2 above)
Comment (luzpaz) [reply src/lib/OpenEXRCore/internal_rle.c]: Added + 2 more related fixed [02fa272](https://github.com/AcademySoftwareFoundation/openexr/pull/1272/commits/02fa2723aa50f5a5fb1cb02a06d68f5b69bebb82)

**Comment by luzpaz** [comment]
@cary-ilm JFYI, you could also add it as a github action: https://github.com/codespell-project/actions-codespell

**Comment by cary-ilm** [comment]
Apologies for the annoyance, but that commit needs the "signed-off-by" line similar to the previous one, could you resubmit? 

And I'll look into adding codespell as an action, thanks.

**Comment by luzpaz** [comment]
My bad, commit has been signed.

**Review by cary-ilm — APPROVED** [review]
Thanks!

**Comment by lgritz** [comment]
Yeah, thanks for this. It has inspired me to try codespell and fix some egregious errors in some other code bases as well. :-)

**Pull request merged**